### DISCUSSION TITLE
Ecommerce API Product stores/returns categorization in name format only

### DISCUSSION THREAD

**Discussion by jsmith2010** [codatio/codat-docs#513] [Q&A]
[https://docs.codat.io/commerce-api#/schemas/Product](url)
The Ecommerce API stores/returns the categorization of a product as a string for the name only. With the example "Hardware"
But the Product Category in Ecommerce API uses a tree structure of child branches with categorization Ids at each level.

If a product is placed in the categorization of "Accessories" as a leaf to more than one category structure, 
such as 
Office Furniture > Accessories or 
Arts and Crafts -> Accessories

A product with the string categorization of "Accessories" would not have enough information to know which higher category it came from. ie was it Office Furniture or Arts and Crafts.

**Comment by mcclowes** [comment]
For reference, which integration(s) are you pulling data from? (e.g. Shopify/Stripe)

**Reply by jsmith2010** [reply]
I have been previewing your API for the assessment of using the ECommerce and Payment as integration into our system. I was interested in how the information is returned/stored and how the categorization is referenced between the different API methods. The use of one field in one method but not another caused me to raise this.
